doc/making-a-release

```diff
@@ -24,7 +24,8 @@ version number (e.g. `/home/geany/publish-release 1.36`).
 Make sure the public part of the GPG key used to sign the archives
 can be found on the download page.
 
-Windows builds (enrico?).
+Windows builds are created mostly by Enrico following the instructions on
```

Comment:
Where do I download the "Enrico" application, or can one be distributed with each Geany? :grin:

Reply:
Ha, I wouldn't recommend that. It's already a few decades around but still buggy from time to time. Also it needs lots of resources, mainly water and food to work properly and somtimes even do weird output.

Not sure I would recommend this application.

Reply:
On a more serious note, I hope we will update this part again in the future.

Recently I had to solve the first quirk with running current MSYS2 packages on my old Windows 7 VM which is not supported anymore.
As stated before, I will *not* upgrade to any newer Windows version ever.

So, either we find another lost soul to have fun with Windows or we use the CI build scripts to create release builds (which is entirely possible already).